badge-maker/README.md

```diff
@@ -1,42 +1,41 @@
-# gh-badges
+# badge-maker
 
-[![npm version](https://img.shields.io/npm/v/gh-badges.svg)](https://npmjs.org/package/gh-badges)
-[![npm license](https://img.shields.io/npm/l/gh-badges.svg)](https://npmjs.org/package/gh-badges)
+[![npm version](https://img.shields.io/npm/v/badge-maker.svg)](https://npmjs.org/package/badge-maker)
+[![npm license](https://img.shields.io/npm/l/badge-maker.svg)](https://npmjs.org/package/badge-maker)
 
 ## Installation
 
 ```sh
-npm install gh-badges
+npm install badge-maker
 ```
 
 ## Usage
 
 ### On the console
 
 ```sh
-npm install -g gh-badges
-badge build passed :green .png > mybadge.png
+npm install -g badge-maker
+badge build passed :green .svg > mybadge.svg
 ```
 
 ### As a library
 
 ```js
-const { BadgeFactory } = require('gh-badges')
-
-const bf = new BadgeFactory()
+const { makeBadge, ValidationError } = require('badge-maker')
```

Comment:
is there a need to import `ValidationError` here?

Reply:
I think we do need to make it clear somewhere in the README that you can import and handle `ValidationError` from the module. I've pushed another commit amending the example a bit so we actually do something with `ValidationError` (and incorporating your other suggestions). Do you think that is a bit clearer?

Reply:
Yup, like it!